bin/perseo

```diff
@@ -76,8 +76,12 @@ function loadConfiguration() {
         config.endpoint.port = process.env.PERSEO_ENDPOINT_PORT;
     }
     if (process.env.PERSEO_MONGO_HOST) {
-        config.mongo.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/cep';
-        config.orionDb.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/orion';
+        var mongoDBEndpoint = process.env.PERSEO_MONGO_HOST;
```

Comment:
We will change documentation to reflect changes

With the change PERSEO_MONGO_HOST to PERSEO_MONGO_ENDPOINT, then provoke other PR in IOT onpremise deployment repository. We do?

With The only think that perseo code should do is to add the /cep and /orion prefixes, consulting on: https://mongodb.github.io/node-mongodb-native/driver-articles/mongoclient.html, if we not provide the port, assumes the default, then we change this without problem (the variable and if)
But the mongodb:/ is better to continue adding in code because all other nodejs telefonicaid repositories do the same